.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.DS_Store
+config.json
```

Comment:
Is there any reason that we need to ignore config.json file? Thanks

Reply:
Config file has sensitive information like AWS_SECRET_KEY. I created a template file as a placeholder